src/test/java/com/g4s8/ghman/bot/UnreadThreadsTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019 Kirill Ch. (g4s8.publci@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.ghman.bot;
+
+/**
+ * Test for {@link Notification.UnreadThreads}.
+ * @since 1.0
+ * @todo #9:30min This test should be implemented after
+ *  https://github.com/g4s8/teletakes/issues/14 is resolved. Please, try to
+ *  avoid using mocked objects, use fake implementations instead.
+ */
+public class UnreadThreadsTest {
```

Comment:
@olenagerasimova class should be final and maybe it is best to keep this todo with `UnreadThreads` instead of introducing this empty file.